### PR TITLE
feat(ios): update babel

### PR DESCRIPTION
**Pull request by m1ga**
Same as https://github.com/tidev/hyperloop.next/pull/370 but for iOS.

not tested as I don't have my Mac with me at the moment

### CODE REVIEW

**Comment by hansemannn** [comment]
Can you make the actions pass? The code itself looks fine!

**Comment by m1ga** [comment]
it builds locally. I'll check it

**Comment by m1ga** [comment]
running out of ideas. `#error unsupported Swift architecture` when trying to build the iOS version. Any idea @hansemannn ? I've tried some of the config from https://github.com/tidev/titanium-sdk/blob/master/.github/workflows/build.yml

**Comment by m1ga** [comment]
Android action is fixed :+1:

**Comment by hansemannn** [comment]
The github action uses an older Xcode version that is not supported by the latest Ti SDK anymore. It's hard to debug, as I have a beta version of macOS locally, but if you align the Ti SDK version with the Xcode version like for other modules, it should work. Otherwise, we'd need to wait until macOS Sequoia (15.0) GA is out, so we can migrate to these actions directly and test them.

File: .github/workflows/ios.yml [hunk 1]
```diff
@@ -12,12 +12,13 @@ on:
 
 jobs:
   ios:
-    runs-on: macos-latest
+    runs-on: macos-13
```
[the remark this answers is not in the document]
Comment (m1ga) [reply .github/workflows/ios.yml:15]: I've tested some changes from the ti-sdk repo. There we had an issue that lastest is using ARM machines now and it had some issues with it. So I've tested https://github.com/tidev/titanium-sdk/blob/master/.github/workflows/build.yml#L28

[the remark this answers is not in the document]
Comment (hansemannn) [reply .github/workflows/ios.yml]: Okay, the underlaying definitely is that the SDK bundles with Swift 5.8 and the latest runners use 6.0+, causing the binary incompatibility. Mid-to-long-term, we have to update the SDK to use the latest one as well, but for now, using the one that the SDK uses is the correct solution. I still wonder though, why your changes didn't work then.

[the remark this answers is not in the document]
Comment (AbdullahFaqeir) [reply .github/workflows/ios.yml:15]: I have a silicon machine, want me to test if it works, or if you want me to look into it

[the remark this answers is not in the document]
Comment (hansemannn) [reply .github/workflows/ios.yml:15]: Would be great if you can check it as well!

[the remark this answers is not in the document]
Comment (AbdullahFaqeir) [reply .github/workflows/ios.yml:15]: Done e

**Review by AbdullahFaqeir — APPROVED** [review]

**Review by AbdullahFaqeir — APPROVED** [review]

**Review by AbdullahFaqeir — COMMENTED** [review]
Fix running action

**Review by AbdullahFaqeir — COMMENTED** [review]
Updated actions to match new version of hyperloop

**Review by AbdullahFaqeir — APPROVED** [review]

**Pull request merged**